tests/vmi_multus_test.go

```diff
@@ -864,29 +866,43 @@ var _ = Describe("[Serial]SRIOV", func() {
 			waitVmi(vmi1)
 			waitVmi(vmi2)
 
+			waitForVMIGuestAgentReport(virtClient, vmi1.Namespace, vmi1.Name, len(vmi1.Spec.Domain.Devices.Interfaces))
+			waitForVMIGuestAgentReport(virtClient, vmi2.Namespace, vmi2.Name, len(vmi2.Spec.Domain.Devices.Interfaces))
+
 			vmi1, err = virtClient.VirtualMachineInstance(vmi1.Namespace).Get(vmi1.Name, &metav1.GetOptions{})
 			Expect(err).NotTo(HaveOccurred())
 			vmi2, err = virtClient.VirtualMachineInstance(vmi2.Namespace).Get(vmi2.Name, &metav1.GetOptions{})
 			Expect(err).NotTo(HaveOccurred())
 
+			vmi1NetworkName := getInterfaceNetworkNameByMAC(vmi1, vmi1.Spec.Domain.Devices.Interfaces[1].MacAddress)
+			Expect(vmi1NetworkName).To(Equal(networkName))
+			vmi2NetworkName := getInterfaceNetworkNameByMAC(vmi2, vmi2.Spec.Domain.Devices.Interfaces[1].MacAddress)
+			Expect(vmi2NetworkName).To(Equal(networkName))
+
 			// manually configure IP/link on sriov interfaces because there is
 			// no DHCP server to serve the address to the guest
 			Expect(configureInterfaceStaticIPByMAC(vmi1, mac1.String(), cidrA)).To(Succeed())
 			Expect(configureInterfaceStaticIPByMAC(vmi2, mac2.String(), cidrB)).To(Succeed())
 
+			return vmi1, vmi2
+		}
+
+		pingThroughSriov := func(vmi1, vmi2 *v1.VirtualMachineInstance, cidrA, cidrB string) {
```

Comment:
I do not see why we need this wrapper now, you can just call the assert & ping from within the test.

Reply:
it repeat twice, still no need ?